src/resolution/graph.rs

```diff
@@ -608,27 +630,16 @@ impl Graph {
       .keys()
       .map(|node_id| (*node_id, self.get_npm_pkg_id(*node_id)))
       .collect::<HashMap<_, _>>();
-    let mut copy_index_resolver =
-      SnapshotPackageCopyIndexResolver::from_map_with_capacity(
-        self.packages_to_copy_index,
-        self.nodes.len(),
-      );
-    let mut packages: HashMap<NpmPackageId, NpmResolutionPackage> =
-      HashMap::with_capacity(self.nodes.len());
+    let mut packages: Vec<NpmResolutionPackage> =
+      Vec::with_capacity(self.nodes.len());
     let mut packages_by_name: HashMap<PackageName, Vec<_>> =
       HashMap::with_capacity(self.nodes.len());
-
-    // todo(dsherret): there is a lurking bug within the peer dependencies code.
-    // You can see it by using `NodeIds` instead of `NpmPackageIds` on this travered_ids
-    // hashset, which will cause the bottom of the "tree" nodes to be populated in
-    // the result instead of the top of the "tree". I think there's maybe one small
-    // thing that's not being updated properly.
```

Comment:
This is not so relevant anymore I think. Let's just remove it and fix bugs when they're reported.